aten/src/ATen/code_template.h

```diff
@@ -99,7 +98,7 @@ struct TemplateEnv {
 # if this list is not empty and ${foo,} will insert one after.
 */
 struct CodeTemplate {
-  /* implicit */ CodeTemplate(std::string t) : template_text(std::move(t)) {}
+  /* implicit */ CodeTemplate(std::string_view t) : template_text(t) {}
```

Comment:
Hmm, now you can use std::move to pass by value though? Is this really an improvement?

Reply:
Oh I see it's only passed by lvalue? probably find then.

Reply:
The call site uses std::string_view.